gadgets/cmr/config/BinningCine/CMR_2DT_RTCine_KspaceBinning_Cloud.xml

```diff
@@ -15,73 +15,41 @@
         Email: hui.xue@nih.gov
     -->
```

Comment:
Since we don't have distributed support, we should remove these "cloud" configurations (pending some other implementation).